plugins/ci/version.txt

```diff
@@ -1 +1 @@
-1.6.2
```

Comment:
probably make this minor release... since the support for this in the backend is not released yet.

Reply:
1.7.0?

Reply:
Bumped to 2.0.0